src/main/java/hello/HelloWorldHttpController.java

```diff
@@ -16,6 +16,8 @@ public String greeting(@RequestParam(value="name", required=false, defaultValue=
         if (true){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "true" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=squid%3AS2583)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this if statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=squid%3AS1145)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)